engine/src/cython/engine.cpp

```diff
@@ -218,18 +216,18 @@ std::unique_ptr<ResultSet> runSkipData(int32_t masterIndex,
 		// Execute query
 		// 		skipdata_output_t
 		//TODO: fix this @alex, input_loaders[0]
-		auto row_groups_cols = ral::skip_data::process_skipdata_for_table(input_loaders[0], minmax_metadata_tables[0], query, queryContext);
-
-		std::vector<cudf::column *> columns;
-		std::vector<std::string> names;
-		for(int i = 0; i < row_groups_cols.size(); i++) {
-			auto& column = row_groups_cols[i];
-			columns.push_back(column.get_gdf_column());
-			// TODO percy cudf0.12 port to cudf::column
-			// GDFRefCounter::getInstance()->deregister_column(column.get_gdf_column());
-
-			names.push_back(column.name());
-		}
+		// auto row_groups_cols = ral::skip_data::process_skipdata_for_table(input_loaders[0], minmax_metadata_tables[0], query, queryContext);
```

Comment:
porque no se necesita?

Reply:
just commenting it out to avoid compile errors. I think @williamBlazing is working on skip data migration